applications/nrf_desktop/src/events/ble_event.h

```diff
@@ -58,6 +58,24 @@ struct ble_qos_event {
 APP_EVENT_TYPE_DECLARE(ble_qos_event);
 #endif
 
+/** @brief Peer types (position in bitmask). */
+enum bt_app_id_type {
+	BT_APP_ID_TYPE_UNUSED,
+	BT_APP_ID_TYPE_GENERAL,
+	BT_APP_ID_TYPE_DONGLE,
+	BT_APP_ID_TYPE_ERASE,
```

Comment:
`BT_APP_ID_TYPE_ERASE_ADV` - to emphasize that the identity is used only for erase advertising procedure